verticapy/__init__.py

```diff
@@ -82,6 +85,7 @@
     "interactive": False,
     "count_on": False,
     "footer_on": True,
+    "identifier": random.randint(0, 1000000000),
```

Comment:
Are the identifiers supposed to be unique?
If yes, you'd better to use Python's UUID module (https://docs.python.org/3.8/library/uuid.html) or UUID generator on Vertica server (https://www.vertica.com/docs/12.0.x/HTML/Content/Authoring/SQLReferenceManual/Functions/UUID/UUID_GENERATE.htm).

Reply:
Good idea, I will use it.

Reply:
Done